Source/WebCore/accessibility/AccessibilityRenderObject.cpp

```diff
@@ -2374,7 +2374,7 @@ VisiblePosition AccessibilityRenderObject::visiblePositionForPoint(const IntPoin
     if (!renderView)
         return VisiblePosition();
 
-#if PLATFORM(COCOA)
+#if PLATFORM(MAC)
     FrameView* frameView = &renderView->frameView();
```

Comment:
There's no real chance of a meaningful side effect here, is there?

Reply:
I don't think so, unless `frameView()` changes state. @cdumez, any thoughts?